src/event_sources/ble_esp32.cc

```diff
@@ -89,7 +87,6 @@ void BLEEventSource::entry() {
         nimble_port_run();
       }
 
-      nimble_port_freertos_deinit();
```

Comment:
Is this not needed anymore?

Reply:
Yeah, it turns out that this isn't needed.